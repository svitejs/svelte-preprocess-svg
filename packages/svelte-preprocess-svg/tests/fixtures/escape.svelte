<script>
	const foo = 'foo';
</script>

<div>
	<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
		<text x="0" y="0">`</text><!-- backtick that would end the @html early -->
		<text x="0" y="8">{foo}</text><!-- curlies could mess with @html -->
		<text x="0" y="16">}</text><!-- another curly -->
	</svg>
</div>
