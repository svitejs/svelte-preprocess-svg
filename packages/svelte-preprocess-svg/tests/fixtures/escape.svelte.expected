<script>
	const foo = 'foo';
</script>

<div>
	<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">{@html `<text>&#96;</text><text y="8">{foo}</text><text y="16">}</text>`}</svg>
</div>
